docs/guides/getting-started.md

```diff
@@ -6,30 +6,45 @@ terraform {
   required_providers {
     checkly = {
       source = "checkly/checkly"
-      version = "1.2.0"
+      version = "1.3.0-rc1"
     }
   }
 }
 
 variable "checkly_api_key" {
 }
 
+variable "checkly_account_id" {
+}
+
 provider "checkly" {
   api_key = var.checkly_api_key
+  account_id = var.checkly_account_id
 }
 ```
 
-2. To use the provider with your Checkly account, you will need an API Key for your Checkly user. Go to the [User Settings: API Keys page](https://app.checklyhq.com/settings/user/api-keys) and click 'Create API Key'. Get your api key and add it to your env `export TF_VAR_checkly_api_key=XXXXXX`
+2. To use the provider with your Checkly account, you will need an API Key for your Checkly user. Go to the [User Settings: API Keys page](https://app.checklyhq.com/settings/user/api-keys) and click 'Create API Key'. Get your User API Key and add it to your env:
+```bash
+$ export TF_VAR_checkly_api_key=cu_xxx
+```
+
+3. You also need to set your target Account ID, you can find the Checkly Account ID [here](https://app.checklyhq.com/settings/account/general).
```

Comment:
I believe the account ID is not visible for people who are read_only for an account.  Maybe this could be explained here?

Reply:
Updated, LMK if is ok now